vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc7519/JsonWebToken.kt

```diff
@@ -0,0 +1,6 @@
+package at.asitplus.wallet.lib.data.rfc7519.jwt
+
+import kotlin.jvm.JvmInline
+
+@JvmInline
+value class JsonWebToken(val value: String)
```

Comment:
Can't we use `JsonWebToken` from signum?

Reply:
removed